samples/android/build.gradle.in

```diff
@@ -90,5 +91,14 @@ gradle.afterProject { project ->
             }
         }
 
+        // Android Gradle Plugin (AGP) 3.5+ is required
+        // https://github.com/android/ndk-samples/wiki/Configure-NDK-Path
+        def isNdkVersionSupported = project.android.metaClass.getProperties().find { it.name == 'ndkVersion' } != null
+        if ((false || opencv_strict_build_configuration) && isNdkVersionSupported) {
```

Comment:
why do you need false here?

Reply:
This option may be useful on the user side (this is a part of OpenCV Android SDK), but it is not enabled by default (to keep Android "defaults" - which depends on versions of user's Android environment).

To manually enable this option just switch `false->true`.